app/models/user.rb

```diff
@@ -50,4 +50,5 @@ def forget
   def downcase_email
     email.downcase!
   end
+  before_save { email.downcase! }
```

Comment:
```suggestion
  before_save :downcase_email
``` and move above function downcase_email